ChangeLog.md

```diff
@@ -21,6 +21,9 @@ See docs/process.md for more on how version tagging works.
 3.1.46 (in development)
 -----------------------
 - libunwind updated to LLVM 16.0.6. (#20088)
+- The `--minify=0` commnad line flag will now preserve comments as well as
+  whitespace.  This means the resulting output can then be run though closure
+  compiler or some other tool that gives comments semantic meaning.
```

Comment:
```suggestion
  compiler or some other tool that gives comments semantic meaning (and
  should have no downside, as `--minify=0` is only used in debug builds anyhow).
```

Reply:
I'm not sure about that though... `--minify=0` isn't just for debug builds.. it might be useful in release builds too, if folks want to do their own minification later on the pipeline.

Reply:
Yeah, maybe you're right. I just think it would be good to explain why we don't think this will inconvenience existing users. So it's either a debug build, or one that the user will minify later anyhow, and either way we aren't hurting anything.

Reply:
Yeah .. I guess we could say something like.. "which is in keeping the with spirit of minify=0 which is designed for folks who either want to debug their code to perform their own minification".   I'm not sure its worth the extra sentence though.